File: php/json_strings.php
<?

$playerSchema = <<<'JSON'
{
  "type": "object",
  "properties": {
    "winPercent": {
      "type": [
        "number",
        "null"
      ]
    },
    "name": {
      "type": "string"
    }
  }
}
JSON;

$goodJson = '{
  "name": "Nooby McNooberson",
  "winPercent": null
}';

$gladJson = '{
  "name": "Mikhail Tal",
  "winPercent": 85.2
}';

$sadJson = '{
  "name": "Nooby McNooberson",
  "winPercent": false
}';

$badJson = '{
  \'name\': "Nooby McNooberson",
  "winPercent": null
}';

$radJson = '{
  "zap": 123,
  "zub": null,
  "ziz": "Nooby McNooberson"
}';

File: c/json_strings.c
// No heredocs in C, but string concatenation is easy.

const char * player_schema = "{"
  "\"name\": null,"
  "\"winPercent\": null"
"}";

const char * good_json = "{"
  "\"name\": \"Nooby McNooberson\","
  "\"winPercent\": null"
"}";

const char * glad_json = "{"
  "\"name\": \"Mikhail Tal\","
  "\"winPercent\": 85.2"
"}";

const char * sad_json = "{"
  "\"name\": \"Nooby McNooberson\","
  "\"winPercent\": false"
"}";

const char * bad_json = "{"
  "\"name\": \"Nooby McNooberson\","
  "'winPercent': null"
"}";

const char * rad_json = "{"
  "\"zip\": 123,"
  "\"zop\": null,"
  "\"zub\": \"Boom!\""
"}";
